runtime/doc/builtin.txt

```diff
@@ -4261,6 +4262,20 @@ getreginfo([{regname}])					*getreginfo()*
 		Can also be used as a |method|: >
 			GetRegname()->getreginfo()
 
+getregion({pos1}, {pos2}				*getregion()*
+		The result is a list of strings where each item is a line
+		from {pos1} to {pos2}.  {pos1} and {pos2} are swapped if
```

Comment:
Returns the list of strings from {pos1} to {pos2}.

For possible values of {pos1} and {pos2} see |line()|.

Useful to get text starting and ending in different columns, such as |characterwise-visual| selection.

Note that 
    - if |virtualmode| is enabled and selection is past the end of line, result lines are filled with spaces,
    - if selection starts or ends in the middle of multiwidth character (e.g. <tab>), it is not included but it's selected part is substituted with spaces.

Example of getting text of visual selection:
....
Example of getting text of visual-block selection:
....
Example of getting text from arbitrary pos1 and pos2
...
Example of getting text from mark1 and mark2
...

Reply:
I have updated.